android/library/src/main/java/com/liferay/mobile/screens/base/interactor/BaseInteractor.java

```diff
@@ -14,6 +16,7 @@
 public abstract class BaseInteractor<L, E extends BasicEvent> implements Interactor<L> {
 
 	protected L listener;
+	protected boolean retried;
```

Comment:
Ok... then we have a problem with rotation changes, but we have to refactor all of this eitherway